algorand.cpp

```diff
@@ -187,6 +187,9 @@ bool is_present(StateSchema schema) {
 bool is_present(Transaction) {
   return true;
 };
+bool is_present(Subsig subsig) {
```

Comment:
I think this is the right idea, but I'd like to see a test of a "threshold" signature - one in which, say, 2 out of 3 signatures are required.  We want to confirm that we are properly including all of the signers' pub keys, even if we have only the two signers' signatures.  That would be an important test to add (in the "golden" format, see comments below).

Reply:
This will be tough since the subsig itself has no knowledge of how many signatures there should be. I'll think about how to add this, however, what should the behavior be when not all the signer's public signatures are present should it throw a runtime error or just return a false?